src/apim_buyerbanks.tf

```diff
@@ -0,0 +1,72 @@
+##############
+## Products ##
+##############
+
+module "apim_buyerbanks_product" {
+  count  = var.buyerbanks_enabled ? 1 : 0
```

Comment:
I think we are going to use too often this count with <something>_enabled
If the resource needs to be created in all environment just create  it.
As far as, I know it's not a best practice to use count everywhere